src/Routes/invest_routes.php

```diff
@@ -13,76 +13,61 @@
 
 $invest = new RouteCollection();
 
-
 ////// INVEST //////
-///
 /// /project/project-name/invest should be the same as /invest/project-name
 $invest->add('invest', new Route(
     '/{project_id}',
-    array('_controller' => 'Goteo\Controller\InvestController::selectRewardAction',
-        )
+    array('_controller' => 'Goteo\Controller\InvestController::selectRewardAction',)
 ));
 
-
 /// This is for compatibility with old routes
 $invest->add('invest-old-route', new Route(
     '/project/{project_id}/invest',
-    array('_controller' => 'Goteo\Controller\InvestController::selectRewardAction',
-        )
+    array('_controller' => 'Goteo\Controller\InvestController::selectRewardAction',)
 ));
 $invest->add('invest-select-payment', new Route(
     '/{project_id}/payment',
-    array('_controller' => 'Goteo\Controller\InvestController::selectPaymentMethodAction',
-        )
+    array('_controller' => 'Goteo\Controller\InvestController::selectPaymentMethodAction',)
 ));
 
 // Custom login for invest process
-$routes->add('invest-login', new Route(
+$invest->add('invest-login', new Route(
     '/invest/{project_id}/login',
-    array('_controller' => 'Goteo\Controller\InvestController::loginAction',
-        )
+    array('_controller' => 'Goteo\Controller\InvestController::loginAction',)
 ));
-$routes->add('invest-signup', new Route(
+$invest->add('invest-signup', new Route(
     '/invest/{project_id}/signup',
-    array('_controller' => 'Goteo\Controller\InvestController::signupAction',
-        )
+    array('_controller' => 'Goteo\Controller\InvestController::signupAction',)
```

Comment:
@javicarrillo could you confirm that this was an actual issue? I found out, after a tidy up of the routes files, that the `invest-login` and `invest-signup` endpoints didn't seems to be working at all. I've renamed the `$routes` variable into `$invest`, and now they should be working again.

Reply:
Hi @carlos-mg89, it worked with the $routes. Since the file was being included directly inside the routes.php, the $routes variable was correct. By changing it to the $invest router, we have to change the route from '/invest/{project_id}/signup' to '/{project_id}/signup'. 

Thanks for pointing this up! I do it myself rn.